Schema generation (#106)
mirumee/ariadne-codegen: README.md

Example with simple schema and few queries and mutations is available [here](https://github.com/mirumee/ariadne-codegen/blob/main/EXAMPLE.md).

## Generating graphql schema's python representation

> Instead of generating client, you can generate file with graphql schema's python representation. To do this provide `graphqlschema` as an argument:

Reviewer: ```suggestion
Instead of generating client, you can generate file with a copy of GraphQL schema as `GraphQLSchema` declaration. To do this call `ariadne-codegen` with `graphqlschema` argument:
```

```
ariadne-codegen graphqlschema
```

> With this strategy command reads configuration from the same place as in [client strategy](#configuration) but supports only `schema_path`, `remote_schema_url`, `remote_schema_headers` options and handles extra ones:  

Reviewer: ```suggestion
`graphqlschema` mode reads configuration from the same place as [`client`](#configuration) but uses only `schema_path`, `remote_schema_url`, `remote_schema_headers` options with addition to some extra options specific to it:  
```

> - `target_file_path` (defaults to `"schema.py"`) - path where to generate file

Reviewer: ```suggestion
- `target_file_path` (defaults to `"schema.py"`) - destination path for generated file
```

- `schema_variable_name` (defaults to `"schema"`) - name for schema variable, must be valid python identifier
- `type_map_variable_name` (defaults to `"type_map"`) - name for type map variable, must be valid python identifier

Generated file contains:

- Necessary imports

> - Type map assignment `{type_map_variable_name}: TypeMap = {...}`

Reviewer: ```suggestion
- Type map declaration `{type_map_variable_name}: TypeMap = {...}`
```

> - Schema assignment `{schema_variable_name}: GraphQLSchema = GraphQLSchema(...)` 

Reviewer: ```suggestion
- Schema declaration `{schema_variable_name}: GraphQLSchema = GraphQLSchema(...)` 
```
